CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 1.2.2 (August 25, 2021)
```

Comment:
I would say that this would be a minor version update, not a patch version update.

Reply:
Done!